source/android/Renaissance/lib/http_services.dart

```diff
@@ -181,6 +181,18 @@ class HttpService {
     return false;
   }
 
+  Future<bool> finish_path(String username, String id) async {
+    String url = baseUrl + '/path/finish-path/';
+    Response res = await post(Uri.parse(url),
+        headers: headers,
+        body:
+        jsonEncode({'username': username, 'path_id': id}));
+    if (res.statusCode == 200) {
+      return true;
+    }
+    return false;
```

Comment:
return response.statusCode == 200;

Reply:
Wow, nice point Kürşat. Another efficient way of coding here!

Reply:
Changes have been made, thanks for warning.